hazelcast/src/main/resources/cluster.sh

```diff
@@ -131,6 +131,33 @@ if [ "$OPERATION" = "change" ]; then
     exit 0
 fi
 
+if [ "$OPERATION" = "force-start" ]; then
+    echo "You are force starting the cluster. Please make sure you provide valid user/pass."
```

Comment:
`force-start makes cluster operational if cluster start is blocked by problematic members.` could be an option